### DISCUSSION TITLE
ca/blog/publica-un-post-a-gironajs

### DISCUSSION THREAD

**Discussion by giscus** [gironajs/gironajs.com#23] [Announcements]
# ca/blog/publica-un-post-a-gironajs

Presentació del projecte GironaJS, la primera comunitat catalana de JavaScript

http://localhost:3000/ca/blog/publica-un-post-a-gironajs

**Comment by eudago** [comment]
test

**Reply by llorenspujol** [reply]
molt xulo aixo dels comments!!

**Comment by llorenspujol** [comment]
test 2

**Reply by sergiogc9** [reply]
Provant cosetes!
